tools/doc/type-parser.mjs

```diff
@@ -49,6 +49,8 @@ const customTypesMap = {
     'worker_threads.html#class-broadcastchannel-' +
     'extends-eventtarget',
 
+  'CallSite': 'https://v8.dev/docs/stack-trace-api#customizing-stack-traces',
```

Comment:
Once this lands, can you ping me, so that I update https://github.com/nodejs/api-docs-tooling/blob/main/src/constants.mjs#L126